### PR TITLE
Travis matrix

### PR DESCRIPTION
**Pull request by caroleolivier**

### CODE REVIEW

**Comment by caroleolivier** [comment]
I new new reviewers...

File: _posts/2017-10-31-building-multiple-project-with-travis.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - language: csharp
+      mono: none
+      dotnet: 2.0.0
+      before_install:
+        - cd data-api
+      script:
+        - dotnet restore
+        - dotnet build
+```
+Note that [Travis linter](https://lint.travis-ci.org/) can't parse the code and returns a lot of errors. However, it works perfectly as you can see here:
```
Comment (simonproffitt) [inline _posts/2017-10-31-building-multiple-project-with-travis.md]: Do you know why that is? Perhaps worth a comment?

File: _posts/2017-10-31-building-multiple-project-with-travis.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Building multiple projects with Travis
+date: 2017-10-31
+---
+
+A few weeks ago, I wrote a [post](https://caroleolivier.github.io/blog/2017/10/03/continuous-integration-in-github) about using [Travis](https://travis-ci.org/) as Continuous Integration tool for some of my repositories in [GitHub](https://github.com/). I briefly mentioned the [Build Matrix](https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix) feature, anticipating it will come handy in the future. And this day has come ^^
+
+
+I am currently writing an application composed of a web application and a data api. I decided to put all the code in one repo (one could argue I should have put them in different repo but this is not the subject of this post). So the obvious question is how do I build both applications using Travis? It turns out it is super easy thanks to Build Matrix and this life saving StackOverflow [post](https://stackoverflow.com/questions/27644586/how-to-set-up-travis-ci-with-multiple-languages).
```
Comment (caroleolivier) [inline _posts/2017-10-31-building-multiple-project-with-travis.md]: can one write repo? or should it be repository?

Comment (simonproffitt) [reply _posts/2017-10-31-building-multiple-project-with-travis.md]: Repo is fine but informal

**Comment by caroleolivier** [comment]
testing, hi simon @simonproffitt

**Review by simonproffitt — CHANGES_REQUESTED** [review]
Ah! It was pending!!!

**Review by simonproffitt — APPROVED** [review]

**Pull request merged**